modules/rodoo-uninstall-resources.adoc

```diff
@@ -0,0 +1,33 @@
+// Module included in the following assemblies:
+//
+// * nodes/pods/run_once_duration_override/run-once-duration-override-uninstall.adoc
+
+:_content-type: PROCEDURE
+[id="rodoo-uninstall-resources_{context}"]
+= Uninstalling {run-once-operator} resources
+
+Optionally, after uninstalling the {run-once-operator}, you can remove its related resources from your cluster.
+
+.Prerequisites
+
+* You have access to the cluster with `cluster-admin` privileges.
+* You have access to the {product-title} web console.
+* You have uninstalled the {run-once-operator}.
+
+.Procedure
+
+. Log in to the {product-title} web console.
+
+. Remove CRDs that were created when the {run-once-operator} was installed:
+.. Navigate to *Administration* -> *CustomResourceDefinitions*.
+.. Enter `RunOnceDurationOverride` in the *Name* field to filter the CRDs.
+.. Click the Options menu {kebab} next to the *RunOnceDurationOverride* CRD and select *Delete Custom Resource Definition*.
```

Comment:
Shouldn't "Options" be in bold?

Reply:
I think this is going to be up for debate soon, it's come up a few times recently and I can't remember who said they might start a convo on this soon. Technically, "Options" isn't in the UI, so I don't think it should be. The OCP guidelines do have it bold, but the majority of places that use it don't. I think it should be lowercase too, but :wink: will keep it this way for now.